ops/terraform/modules/resources/templates/fhir_server.tpl

```diff
@@ -26,6 +26,6 @@ cat <<EOF >> extra_vars.json
 }
 EOF
 
-ansible-playbook --extra-vars '@extra_vars.json' --vault-password-file=vault.password --tags "post-ami" launch_bfd-server.yml
+ansible-playbook -vvv --extra-vars '@extra_vars.json' --vault-password-file=vault.password --tags "post-ami" launch_bfd-server.yml
```

Comment:
Did you intend to leave playbook debugging on before merging or was this just a temporary thing?

Reply:
I did not and have removed the -vvv flag.